apps/cluster/README.md

```diff
@@ -0,0 +1,30 @@
+## Synopsis
+
+This part of the module is for clustering machines created by Terraform
+When creating the stack, point to this module to use machines with their SG. Plus load balancer and its SG. 
+
+## Code Example
+
+
+```hcl-terraform
+module "consul-cluster" {
+  source = "git@github.com:albumprinter/eops_tf_modules.git//apps/cluster?ref={TAG_VERSION}" # for eg. {TAG_VERSION} = v0.7.0
+    tags_team = "eops" //team name
+    environment = "dev-nonprod-prod" 
+    app_name = "eops-consul" //name of the application for cluster
+    app_elb_name = "eops-consul_ELB" //name of the loadbalancer
```

Comment:
Let's create a follow up case to streamline the names, also we should add in a parameter for custom json chef provisioner!